### PR TITLE
[refactor] Add actions_spec

### PR DESCRIPTION
**Pull request by donaldong**
This fixes some existing rubocop lint errors and move test cases related to actions.rb to a separate file.

### CODE REVIEW

File: spec/actions_spec.rb [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+  context 'create' do
+    it '#create!' do
+      instance = klass.create!(value: 3)
+      another_instance = klass.find(instance.id)
+      expect(instance.value).to eq another_instance.value
+    end
+
+    it 'validates types' do
+      # TODO: this should raise an error
```
Comment (donaldong) [inline spec/actions_spec.rb:39]: See https://github.com/chanzuckerberg/redcord/issues/42

**Comment by donaldong** [comment]
remove unused model attributes in spec

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redcord/compare/2bcec7f51c121583daab18f4b8bbcfd4259fa843..929d77190b442e526aa9e755334befafb73b8068).

File: spec/actions_spec.rb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      migrator.change_ttl_passive(klass)
+      expect(klass.redis.ttl(instance.instance_key)).to eq(-1)
+
+      instance.save!
+      expect(klass.redis.ttl(instance.instance_key) > 0).to be true
+
+      klass.ttl(nil)
+      migrator.change_ttl_passive(klass)
+      instance.update!(value: 4)
+      expect(klass.redis.ttl(instance.instance_key)).to eq(-1)
```
Comment (katyho) [inline spec/actions_spec.rb:116]: `change_ttl_passive` should make a call to redis to change the ttl for the model right? Do we need to call `save!` and `update!` as well?

File: spec/actions_spec.rb [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+  context 'delete' do
+    it '#destroy' do
+      instance = klass.create!(value: 1)
+
+      another_instance = klass.find(instance.id)
+      expect(another_instance.id).to eq instance.id
+      expect(another_instance.value).to eq instance.value
+      expect(another_instance.created_at.to_i).to eq instance.created_at.to_i
+      expect(another_instance.updated_at.to_i).to eq instance.updated_at.to_i
```
Comment (katyho) [inline spec/actions_spec.rb:150]: I feel like these checks should be done in most other tests as well. Maybe we can have a helper method, e.g. `eq_redcord` which checks that all ids and attributes are equal between two redcords.

File: spec/actions_spec.rb (hunk 2 above)
Comment (donaldong) [reply spec/actions_spec.rb:116]: ah, so `change_ttl_passive` only updates the ttl for new records (which have been touched recently). For the ttl to be applied, we also need to touch the record using save! / update! (or we can call change_ttl_active (to be implemented), which automatically touches all existing records)

File: spec/actions_spec.rb (hunk 3 above)
Comment (donaldong) [reply spec/actions_spec.rb:150]: Good idea!

Comment (katyho) [reply spec/actions_spec.rb:150]: This can probably come in a separate change if you want, so approving for now

**Pull request merged**